src/PowerShellEditorServices.Host/LanguageServer.cs

```diff
@@ -3,98 +3,76 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 //
 
-using DebugAdapterMessages = Microsoft.PowerShell.EditorServices.Protocol.DebugAdapter;
 using Microsoft.PowerShell.EditorServices.Protocol.LanguageServer;
 using Microsoft.PowerShell.EditorServices.Protocol.MessageProtocol;
+using Microsoft.PowerShell.EditorServices.Protocol.MessageProtocol.Channel;
 using Microsoft.PowerShell.EditorServices.Utility;
 using Nito.AsyncEx;
 using System;
 using System.Collections.Generic;
+using System.IO;
 using System.Linq;
 using System.Management.Automation;
 using System.Management.Automation.Language;
 using System.Text.RegularExpressions;
 using System.Threading;
 using System.Threading.Tasks;
-using System.IO;
+using DebugAdapterMessages = Microsoft.PowerShell.EditorServices.Protocol.DebugAdapter;
 
 namespace Microsoft.PowerShell.EditorServices.Host
 {
-    internal class LanguageServer : IMessageProcessor
+    internal class LanguageServer : LanguageServerBase 
```

Comment:
Maybe I'm missing intention here but it seems like this class should be public since it implements a bunch of handy message handling stuff and right now it only allows the use StdioServer channel class. I know there aren't any other channels yet but I would think this class should have a constructor that accepts a ChannelBase.